ISSUE 9: Copyedited README

daviserinanderson opened the pull request:
In commit no. a148094, I copyedited README: 
- added or removed hyphens
- ensured common nouns are lower case
- added commas and other punctuation where appropriate
- changed "it" to "Archipelago" in order to disambiguate 
- edited section headings to camelCase

In commit no. a76b220, I edited a typo I made in commit no. a148094.

Commit:
ISSUE 9: Copyediting README document

Copyedited README: 
- added or removed hyphens
- ensured common nouns are lower case
- added commas and other punctuation where appropriate
- changed "it" to "Archipelago" in order to disambiguate 
- edited section headings to camelCase

Commit:
ISSUE 9: Content Edits to README.md

Copyedited README:
- added or removed hyphens
- ensured common nouns are lower case
- added commas and other punctuation where appropriate
- changed "it" to "Archipelago" in order to disambiguate
- edited section headings to camelCase

DiegoPino commented on README.md:
```diff
@@ -1,87 +1,77 @@
 
 
-## Strawberryfields for ever
+## Strawberryfields Forever
```
ha ha! This is a tricky one. So i went for the british version, the two words thing to distinguish it from the song (Even you all know the whole thing comes from there). So reading this http://www.grammarphobia.com/blog/2012/02/forever.html What do you think represents better SBF involvement?

daviserinanderson replied:
Very interesting! I'm learning something new. I'm all for the two-word version. How about "Strawberryfields For Ever" as the section header here? After all, I was seeking to standardize the casing in the section headers more than any other outcome.

DiegoPino commented on README.md:
```diff
@@ -1,87 +1,77 @@
 
-But by design (and because we think its imperative) it takes fully charge, owns, the Metadata layer and their associated Media assets by implementing a highly
-configurable and also smart JSON based Drupal field named [`strawberryfield`](https://github.com/esmero/strawberryfield/tree/8.x-1.0-beta1) that attaches to any Content. 
+By design (and because we think its imperative), Archipelago takes full charge of the metadata layer and associated media assets by implementing a highly configurable, smart JSON-based Drupal field named [`strawberryfield`](https://github.com/esmero/strawberryfield/tree/8.x-1.0-beta1) that attaches to any content. 
```
Here i would like to keep the work JSON separated from the based, is that grammatically incorrect? Not a strong opinion but some buzzwords, core to Archipelago's architecture i feel live better by them selves.

daviserinanderson replied:
I read "JASON-based" as a compound noun; including a hyphen is grammatically correct. Perhaps we could consider alternate phrasing for this. Or we can just ignore the rules. 😉

DiegoPino reviewed (approved):
Hi @daviserinanderson i approve this, still i added a few comments i would love to discuss. Nothing is pressing and it can be merged at it is, its just my playful nature to find the exact right intention of words. You have exceeded all my expectations on this set of Documentation fixes, thank you so much ❤️

Commit:
Address Code Review

Addressing comments from @DiegoPino:
- Changed "Forever" to "For Ever" in a section header 
- Reworded phrase to avoid the term JSON-based (see line 31)

Thanks, @diegopino, for your review! 😄
